src/tests/unit_tests/test_document_converter.py

```diff
@@ -402,7 +412,64 @@ def test_convert_langchain_document(langchain_doc, firestore_doc):
 )
 def test_roundtrip_firestore(firestore_doc):
     langchain_doc = convert_firestore_document(firestore_doc)
-    roundtrip_doc = convert_langchain_document(langchain_doc, pytest.client)
+    roundtrip_doc = convert_langchain_document(langchain_doc, firestore_client)
 
     assert roundtrip_doc["data"] == firestore_doc.to_dict()
     assert roundtrip_doc["reference"]["path"] == firestore_doc.reference.path
+
+
+@pytest.mark.parametrize(
+    "firestore_doc",
+    [
+        (
+            DocumentSnapshot(
+                reference=DocumentReference(*["foo", "bar"], client=firestore_client),
+                data={
+                    "embedding": Vector([1, 2, 3]),
+                    "content": "test_doc2",
+                },
+                exists=True,
+                read_time=None,
+                create_time=None,
+                update_time=None,
+            )
+        ),
+    ],
+)
+def test_vector_type_from_firestore(firestore_doc):
```

Comment:
Is is not possible to create the expected LangChain Document,  and compare the conversion? -- similar to tests above

Instead of asserting individual fields?

Reply:
Fixed